config/environments/development.rb

```diff
@@ -26,6 +26,11 @@
     config.cache_store = :null_store
   end
 
+  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
+  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
+  # `config/secrets.yml.key`.
+  config.read_encrypted_secrets = true
```

Comment:
Should we define this inside `production.rb` as well?

Reply:
Production has it by default